src/thor/dijkstras.cc

```diff
@@ -78,7 +78,7 @@ void Dijkstras::Initialize(label_container_t& labels,
   uint32_t edge_label_reservation;
   uint32_t bucket_count;
   GetExpansionHints(bucket_count, edge_label_reservation);
-  labels.reserve(std::min(max_reserved_labels_count_, edge_label_reservation));
```

Comment:
this used have the effect of capping the reservation at `kInitialEdgeLabelCountDijkstras` now you can configure to allow for more. i guess thats ok, but the diff isnt enough to understand thats happening

Reply:
I can confirm that is what's happening:)